src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckDistributionRestApi.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (C) 2018 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.provider.blackduck.descriptor;
+
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.descriptor.config.RestApi;
+import com.synopsys.integration.alert.database.entity.DatabaseEntity;
+import com.synopsys.integration.alert.web.model.Config;
+
+@Component
+public class BlackDuckDistributionRestApi extends RestApi {
+
+    @Autowired
+    public BlackDuckDistributionRestApi(final BlackDuckTypeConverter typeConverter, final BlackDuckRepositoryAccessor repositoryAccessor) {
+        super(typeConverter, repositoryAccessor);
+    }
+
+    @Override
+    public void validateConfig(final Config restModel, final Map<String, String> fieldErrors) {
+
+    }
+
+    @Override
+    public void testConfig(final DatabaseEntity entity) {
+
```

Comment:
Shouldn't there be a way to test the config here?

Reply:
For testing hub project names what would we test here? This just creates a placeholder for us to perform distribution.  Perhaps based on your question here maybe Provider distributions shouldn't have a way to test since we aren't pushing a message to the provider.